utils.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/python
+#
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+
+def shell(command, timeout=None):
+  try:
+    subprocess.run(command, shell = True, check = True, timeout=timeout)
```

Comment:
nit: [python style](https://www.python.org/dev/peps/pep-0008/#:~:text=don't%20use%20spaces%20around%20the%20%3D%20sign%20when%20used%20to%20indicate%20a%20keyword%20argument%2C%20or%20when%20used%20to%20indicate%20a%20default%20value%20for%20an%20unannotated%20function%20parameter) advises to remove spaces between args and default values so:
```python
subprocess.run(command, shell=True, check=True, timeout=timeout)
```